.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  pull_request:
```

Comment:
Does this need to be there? The job looks to only run on push anyhow

Reply:
What this does it it will make any branch tagged v* e.g. v.1.2.4 generate corresponding image tags. This makes versioned releases and rolling back very easy.

Reply:
This was in reference to the 'pull requests' which are not used since the only job on the workflow checks to ensure that it was a push action. Please remove it since it is unused.

Reply:
Oh yeah, that part doesn't do anything either way since it's an empty key. Removed.

Reply:
Good point, removed